src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -0,0 +1,4224 @@
+        /* Prefix7 */ [|MethodOverloadWithArguments|](5 /* ArgumentComment2 */) /* Suffix7 */;
+        /* Prefix8 */ MethodOverloadWithArguments(x: /*ArgumentComment3 */ 5 /* ArgumentComment4 */, ct) /* Suffix8 */;
+
+    }
+    Task MethodDefaultAsync(CancellationToken c = default) => Task.CompletedTask;
+    Task MethodOverloadAsync() => Task.CompletedTask;
+    Task MethodOverloadAsync(CancellationToken c) => Task.CompletedTask;
+    Task MethodOverloadWithArgumentsAsync(int x) => Task.CompletedTask;
+    Task MethodOverloadWithArgumentsAsync(int x, CancellationToken c) => Task.CompletedTask;
+    void MethodDefault(CancellationToken c = default) {}
+    void MethodOverload() {}
+    void MethodOverload(CancellationToken c) {}
+    void MethodDefaultWithArguments(int x, CancellationToken c = default) {}
+    void MethodOverloadWithArguments(int x) {}
+    void MethodOverloadWithArguments(int x, CancellationToken c) {}
+}
+            ";
+            string fixedCode = @"
+using System.Threading;
+using System.Threading.Tasks;
+class C
+{
+    async void M(CancellationToken ct)
+    {
+        await /* Prefix1 */ MethodDefaultAsync(ct) /* Suffix1 */;
+        await /* Prefix2 */ MethodOverloadAsync(ct) /* Suffix2 */;
+        await /* Prefix3 */ MethodOverloadWithArgumentsAsync(5 /*ArgumentComment0 */, ct) /* Suffix3 */;
+        /* Prefix4 */ MethodDefault(ct) /* Suffix4 */;
+        /* Prefix5 */ MethodOverload(ct) /* Suffix5 */;
+        /* Prefix6 */ MethodDefaultWithArguments(5 /* ArgumentComment1 */, ct) /* Suffix6 */;
+        /* Prefix7 */ MethodOverloadWithArguments(5 /* ArgumentComment2 */, ct) /* Suffix7 */;
+        /* Prefix8 */ MethodOverloadWithArguments(x: /*ArgumentComment3 */ 5 /* ArgumentComment4 */, ct) /* Suffix8 */;
+
+    }
+    Task MethodDefaultAsync(CancellationToken c = default) => Task.CompletedTask;
+    Task MethodOverloadAsync() => Task.CompletedTask;
+    Task MethodOverloadAsync(CancellationToken c) => Task.CompletedTask;
+    Task MethodOverloadWithArgumentsAsync(int x) => Task.CompletedTask;
+    Task MethodOverloadWithArgumentsAsync(int x, CancellationToken c) => Task.CompletedTask;
+    void MethodDefault(CancellationToken c = default) {}
+    void MethodOverload() {}
+    void MethodOverload(CancellationToken c) {}
+    void MethodDefaultWithArguments(int x, CancellationToken c = default) {}
+    void MethodOverloadWithArguments(int x) {}
+    void MethodOverloadWithArguments(int x, CancellationToken c) {}
+}
+            ";
+            return VerifyCS.VerifyCodeFixAsync(originalCode, fixedCode);
+        }
+
+        [Theory]
+        [InlineData("CancellationToken c", "", "", "")]
+        [InlineData("", "CancellationToken c", "", "")]
+        [InlineData("", "", "<CancellationToken>", "c")]
+        public Task CS_Diagnostic_MultiNesting(string topMethodParam, string localMethodParam, string extensionTypeParam, string extensionArg)
```

Comment:
Ah, thanks. Ignore my previous question about a test where outermost has ct, but not inner one.